### ISSUE TITLE
Visual Studio Code create a lot of zombie processes

### ISSUE THREAD

**Issue by Geliozzz** [microsoft/vscode-cpptools#9282]
### Environment

- OS and version: Mac Moteray 12.3.1
- VS Code:1.66.2
- C/C++ extension: v1.10.0 Pre-Release
- OS and version of remote machine (if applicable): Red Hat Enterprise Linux 8.4
- GDB / LLDB version: GNU gdb (GDB) Red Hat Enterprise Linux 8.2-15.el8

### Bug Summary and Steps to Reproduce

Bug Summary:
Visual Studio Code create a lot of zombie processes on the server.
Tasks: 1786 total, 5 running, 1184 sleeping, 32 stopped, 565 zombie
3963842 0.0 0.0 0 0 ? Z Apr25 0:00 [cpptools-srv] 
3965040 0.0 0.0 0 0 ? Z Apr25 0:00 [cpptools-srv] 
3965119 0.0 0.0 0 0 ? Z Apr25 0:00 [cpptools-srv] 
3968225 0.0 0.0 0 0 ? Z Apr25 0:00 [cpptools-srv] 
4016797 0.0 0.0 0 0 ? Z Apr25 0:00 [cpptools-srv] 
4017007 0.0 0.0 0 0 ? Z Apr25 0:00 [cpptools-srv] 
4022034 0.0 0.0 0 0 ? Z Apr25 0:00 [cpptools-srv] 
4037473 0.0 0.0 0 0 ? Z Apr27 0:00 [cpptools-srv] 
4047862 0.0 0.0 0 0 ? Z Apr27 0:00 [cpptools-srv] 
4048254 0.0 0.0 0 0 ? Z Apr25 0:00 [cpptools-srv] 
4048656 0.0 0.0 0 0 ? Z Apr25 0:00 [cpptools-srv] 
4048711 0.0 0.0 0 0 ? Z Apr25 0:01 [cpptools-srv] 
4094212 0.0 0.0 0 0 ? Z Apr27 0:00 [cpptools-srv] 
4141827 0.0 0.0 0 0 ? Z Apr27 0:00 [cpptools-srv] 
4144533 0.0 0.0 0 0 ? Z Apr27 0:00 [cpptools-srv]

Steps to reproduce:
I don't understand how it reproduces exactly, but after some time I see in system a lot of zombie process.

### Debugger Configurations

```shell
{
    "tasks": [
        {
            "type": "cppbuild",
            "label": "C/C++: g++ build active file",
            "command": "/usr/bin/g++",
            "args": [
                "-fdiagnostics-color=always",
                "-g",
                "${file}",
                "-o",
                "${fileDirname}/${fileBasenameNoExtension}"
            ],
            "options": {
                "cwd": "${fileDirname}"
            },
            "problemMatcher": [
                "$gcc"
            ],
            "group": {
                "kind": "build",
                "isDefault": true
            },
            "detail": "Task generated by Debugger."
        }
    ],
    "version": "2.0.0"
}
```

### Debugger Logs

```shell
The problem is not in debug mode
```

**Comment by tustin2121** [comment]
I second this bug. I'm on Windows 10, and in my Process Explorer tree, orphaned at the bottom of the list, there can be dozens of these things which I have to manually kill every so often. I'm not sure what generates them, as restarting the extension host or the window makes one that gets cleaned up, but using VSCode throughout the day or even leaving it overnight and I'll come back later to a dozen zombie ones.
![image](https://user-images.githubusercontent.com/794812/167181930-4120b434-3edf-43c8-a855-0fa2356fef68.png)

These don't seem to be using any CPU. And this doesn't happen on the currently released version, only the preview version v1.10.0.

**Comment by michelleangela** [comment]
Thank you for reporting this. This is being tracked by existing issue https://github.com/microsoft/vscode-cpptools/issues/9261.

Closing as a duplicate.
